client/src/components/Ticketing/ticketingmanager/Refund/RefundOrderTicketMain.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Udash_nav from '../udash_navbar';
+import Refund from './RefundOrderTicket';
+
+/**
+ * Hosts refund page
+ *
+ * @returns {Navbar, RefundPage, Footer}
+ */
+const RefundMain = () => {
+  return (
+    <div className='flex flex-row'>
+        <Udash_nav/>
+        <Refund/>
```

Comment:
Not too important but I'm getting ESlint warnings here regarding spacing. Do you have prettier setup?

Reply:
Actually... no, I have "Prettier - Code formatter v10.1.0", but not "Prettier ESLint", I'll get that installed!